app/assets/stylesheets/nav.scss

```diff
@@ -45,7 +45,7 @@ body{
 
 body.home {
   background-image: url("http://static4.businessinsider.com/image/53a1eb91ecad045e6987bc69-1200/its-hard-to-say-whether-this-is-really-an-office-or-just-a-hip-cafe.jpg");
-  background-size: 150em;
+  background-size: 100em;
```

Comment:
Properties should be ordered background-image, background-repeat, background-size